auth/casbin/middleware.go

```diff
@@ -0,0 +1,65 @@
+package casbin
+
+import (
+	"context"
+	"errors"
+
+	stdcasbin "github.com/casbin/casbin"
+	"github.com/go-kit/kit/endpoint"
+)
+
+type contextKey string
+
+const (
+	// CasbinModelContextKey holds the key to store the access control model
+	// in context, it can be a path to configuration file or a casbin/model
+	// Model
```

Comment:
Still need punctuation etc. on all doc comments like this one.

Reply:
submitted and update, would it be like that ? sorry kinda new into this.

https://github.com/go-kit/kit/pull/787/commits/bf51e5ee889a96124f7d81177a53b9bb5ce56d64